.github/workflows/docs.yml

```
@@ -82,7 +82,7 @@ jobs:
             --pipeline chembl_activity \
             --check
 
-      - name: Check generated package-family class diagram drift
+      - if: ${{ false }} # Full-corpus package-family generation is disabled.
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)